### DISCUSSION TITLE
Watch list/Rarity/Protection status

### DISCUSSION THREAD

**Discussion by PovilasID** [tphakala/birdnet-go#170] [Ideas]
I would love to get a notification if a rare/protected bird or something from my list was detected.
You can totally get notification on detection using MQTT and Home Assistant or Gotify, so I am just missing a tag in MQTT to filter the notifications. I could be something like: `status: endangered` or `lists: ["protected"; "owls"]` .
I have list of protected species latin names in my area, so I can just add the list myself, however, considering an app has my location and rarity is subject to location and this information should be available it could probably pull down some list from some source though I am not sure what data source could be for that.

P.S. I am aware I can do the filtering on the Home Assistant side and that this project is fairly early in its development, so this a far in the future thing.

**Comment by tphakala** [comment]
Yes this is planned, just so many things to do and so little time ..

**Comment by PovilasID** [comment]
Hey awesome progress for the project.
If you get to this and start wondering that to use as source I have a suggestion:
![image](https://github.com/user-attachments/assets/02b6e04c-9883-4dad-adf3-2e2355334362)
I saw the Wikipedia has 'conservation status' often listed in the info boxes.
Here is also a quick article form wikimedia foundation they now have API for calling structured data from info boxes
https://enterprise.wikimedia.com/blog/structured-contents-wikipedia-infobox/
You can always scrape wikipedia... but that just not very nice :P

**Reply by Shellfishgene** [reply]
IUCN Red List also has an API to get that information directly from them:
https://apiv3.iucnredlist.org/
